api/Navigator.json

```diff
@@ -1426,7 +1427,8 @@
               "version_added": null
             },
             "firefox": {
-              "version_added": "3"
+              "version_added": "3",
+              "notes": "Starting in Firefox 62, this function must only be used in a secure context."
```

Comment:
For consistency with other secure context data, I think what this note is saying should be represented in a `"secure_context_required"` subfeature of `registerProtocolHandler` (a bit [like this](https://github.com/mdn/browser-compat-data/blob/0fd82d0c7e966a44ce89dd14f9e33fb00375041f/html/elements/html.json#L111))